yt/utilities/exceptions.py

```diff
@@ -872,3 +872,19 @@ class GenerationInProgress(Exception):
     def __init__(self, fields):
         self.fields = fields
         super(GenerationInProgress, self).__init__()
+
+
+class YTAmbiguousFieldName(YTException):
+    def __init__(self, fname, possible_ftypes):
+        self.fname = fname
+        self.possible_ftypes = possible_ftypes
+
+    def __str__(self):
+        msg = (
+            f"The requested field name '{self.fname}'"
+            "is ambiguous and corresponds to any one of "
+            f"the following field types\n {self.possible_ftypes}."
+            "Please specify the requested field as an explicit "
+            "tuple (ftype, fname)."
```

Comment:
```suggestion
            f"The requested field name '{self.fname}' "
            "is ambiguous and corresponds to any one of "
            f"the following field types\n {self.possible_ftypes}. "
            "Please specify the requested field as an explicit "
            "tuple (ftype, fname)."
```

Reply:
Oops, indeed I got the spacing wrong it seems.